src/JMS/Serializer/XmlSerializationVisitor.php

```diff
@@ -454,7 +454,7 @@ private function addNamespaceAttributes(ClassMetadata $metadata, \DOMElement $el
 
     private function createElement($tagName, $namespace = null)
     {
-        if (null === $namespace) {
+        if (null === $namespace || $this->document->isDefaultNamespace($namespace)) {
```

Comment:
what you are trying to solve, i tried it with https://github.com/schmittjoh/serializer/pull/644